kf_lib_data_ingest/common/concept_schema.py

```diff
@@ -102,20 +102,32 @@ class DIAGNOSIS(PropertyMixin):
         CATEGORY = None
         UBERON_TUMOR_LOCATION_ID = None
         EVENT_AGE_DAYS = None
+        VERIFICATION = None
+
+        class ABATEMENT_EVENT_AGE(QuantityMixin):
```

Comment:
I think we should have the corresponding _DAYS field with any new age quantities to use with https://github.com/kids-first/kf-lib-data-ingest/blob/92e8b9224a32a82d63e655af25a356c144b4da69/kf_lib_data_ingest/common/misc.py#L18-L31

Reply:
Why though? I thought the other instances of this were for backward compatibility?